legal-api/report-templates/template-parts/common/nameTranslation.html

```diff
@@ -1,5 +1,5 @@
 <div class="no-page-break">
-    {% if listOfTranslations|length == 0 %}
+    {% if listOfTranslations|length or nameTranslations|length %}
```

Comment:
Is there a unit test to verify this fix?

I'm actually a bit confused on why there are 2 lists here. How does it work with the logic on line 7? What are the various scenarios? Can any logic be simplified (or commented) to reduce confusion?

Also, are there any version numbers that need to be updated?

Reply:
It's just a fix for changes that Arturo did to the reports. 

There's no tests but I can add one. The variables are confusing. I'm going to change that as well. Thanks.

Reply:
I added tests. I think it's easier to understand now. What do you think?

Reply:
Awesome! Yes, way better.

I was trying to understand the logic on line 7. I think it's still a bit weird but the unit tests do an excellent job of covering all the cases, so it's fine.